Core/Src/main.c

```diff
@@ -46,9 +46,10 @@
 /* Private variables ---------------------------------------------------------*/
 
 /* USER CODE BEGIN PV */
-static const uint8_t ACTIVATE_ADC_VALUE = 1;
-static const uint16_t MAX_TIMER_VALUE = 64000;
-static const uint16_t MAX_ADC_VALUE = 1024;
+static const uint32_t ACTIVATE_ADC_VALUE = 6144; // Transmitted data to ADC: 000000001 10000000 00000000 => select single ended, channel 0
```

Comment:
I believe if you make the number a decimal this is 98304, I would recommend splitting it up by byte and having a 3 long array of bytes. Then you can assign each byte easier.